apps/dotcom/client/styles/core.css

```diff
@@ -1,4 +1,10 @@
-@import url('https://fonts.googleapis.com/css2?family=Inter:wght@500;700&display=swap');
+/* NB: If we did position: fixed, Safari on iOS 18 and MacOS 15 would
```

Comment:
not the biggest deal but removing the font import here affects the BoardHistorySnapshot (which doesn't pull in globals.css)